tests/playwright/src/utility/fixtures.ts

```diff
@@ -58,5 +60,9 @@ export const test = base.extend<TestFixtures & FixtureOptions>({
     const statusBar = new StatusBar(page);
     await use(statusBar);
   },
+  kubernetesBar: async ({ navigationBar }, use) => {
+    const kubernetesBar = await navigationBar.openKubernetes();
+    await use(kubernetesBar);
+  },
```

Comment:
I'm not sure I understand what the point of this fixture is, the automatic navigation makes it impractical for it to be a fixture.